pontoon/base/models.py

```diff
@@ -2640,6 +2646,7 @@ def map_entities(cls, locale, preferred_source_locale, entities, visible_entitie
                 'source': entity.source,
                 'obsolete': entity.obsolete,
                 'translation': translation_array,
+                'translation_comments': translation_comments_array,
```

Comment:
Same as on frontend: this should be moved to `pontoon.base.views:get_translation_history()`.